zipkin-receiver-kafka/README.md

```diff
@@ -20,6 +20,25 @@ $ KAFKA_ZOOKEEPER=127.0.0.1:2181 bin/collector
 $ KAFKA_ZOOKEEPER=127.0.0.1:2181 KAFKA_TOPIC=notzipkin bin/collector
 ```
 
+### Encoding spans into Kafka messages
```

Comment:
@abesto PS this is the exact encoding of the POST /api/v1/spans endpoint (when content-type is application/x-thrift). Maybe we can move this to the new website and link out to there at some point.